{{ cookiecutter.project_name }}/runtime.txt

```diff
@@ -1 +1 @@
-python-3.5.1
```

Comment:
No technical reason, but it would mean installing it manually since none of the ubuntu docker containers point to Python 3.5 yet